src/Shared/PackageManagers/GitHubProjectManager.cs

```diff
@@ -177,7 +177,7 @@ public static IEnumerable<PackageURL> ExtractGitHubUris(PackageURL purl, string
 
             try
             {
-                foreach (Match match in GithubExtractorRegex.Matches(searchText))
+                foreach (Match match in GithubExtractorRegex.Matches(searchText).Where(match => match != null))
```

Comment:
Is there are reason match could be null?

Reply:
Intellisense has deemed it thus.